src/modules/apps/install.ts

```diff
@@ -116,13 +93,12 @@ const prepareInstall = async (appsList: string[], force: boolean): Promise<void>
         log.error(e.response.data.message)
       } else {
         switch (e.message) {
-          case InstallStatus.USER_HAS_NO_BUY_APP_LICENSE:
```

Comment:
This is not a possibility anymore?

Reply:
Exactly. Since no users will buy apps using this interface alone, we should not need this check anymore.

Reply:
But what if `installApp` returns a `USER_HAS_NO_BUY_APP_LICENSE` error? Would that be possible?

Reply:
Since `vtex.billing@0.22.0`, this status is no longer needed 😁 
https://github.com/vtex/billing-app/commit/2c836d13900af9530668da73ac266f244dacb48c#diff-acd867f3a9c343b754e87e72a47d8253e771fd2c8c4cc43ed358577bcd62124fL76